src/command_modules/azure-cli-cloud/azure/cli/command_modules/cloud/_params.py

```diff
@@ -6,7 +6,7 @@
 
 from azure.cli.core.commands import register_cli_argument
 
-from azure.cli.core.cloud import get_clouds, get_custom_clouds
+from azure.cli.core.cloud import get_clouds, get_custom_clouds, get_active_cloud_name
 
 
 # pylint: disable=line-too-long
```

Comment:
_params.py (and commands.py) are the main cases where we deliberately keep this in. We went back in forth on the single-line/multi-line thing and ultimately went the route of single line, which all but guarantees this Pylint check should be disabled.